tensorflow/contrib/nn/python/ops/modrelu_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for modrelu.py."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import modrelu
+# from tensorflow.contrib.nn.python.ops import modrelu
```

Comment:
Our style is to import the full module paths - is there any reason it didnt work?